cpp/oneapi/dal/backend/interop/data_conversion.cpp

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/backend/interop/data_conversion.hpp"
+#include "oneapi/dal/exceptions.hpp"
+
+using namespace daal::data_management;
```

Comment:
Have looked at latest coding style document: 

> NS9: Do not use user-visible using namespace
directive in any header file. It pollutes the namespace and destroys all
efforts to maintain namespaces. Despite this, using namespace directive is
allowed in function scope or source files.

No restrictions about cpp file. However, I applied your proposal to increase clarity. What's about other usings like `using std::int64_t` in sources?

Reply:
> What's about other usings like using `std::int64_t` in sources?

`using type` is more "safe" than `using namespace`, because you control what is imported.

In general, I'd not use `using namespace  <old DAAL namespaces>` at all even in source files (rarely in function scope). Old DAAL namespaces are polluted, nobody cares what's really exported, they contain a lot of old-fashioned `enums` values  which are exported together with namespace, e.g., `readOnly`, `write`, etc.

I'd like to clearly distinguish old API calls and new, fully qualified names or aliases for namespaces are helpful.